src/main/java/com/icthh/xm/tmf/ms/resource/config/AsyncConfiguration.java

```diff
@@ -17,6 +15,9 @@
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
 import org.springframework.scheduling.config.ScheduledTaskRegistrar;
 
+import java.util.concurrent.Executor;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version